File: sample/src/main/java/contacts/sample/cheatsheet/groups/java/InsertGroupsActivity.java
package contacts.sample.cheatsheet.groups.java;

import android.accounts.Account;
import android.app.Activity;

import java.util.List;

import contacts.core.ContactsFactory;
import contacts.core.entities.NewGroup;
import contacts.core.groups.GroupsInsert;

public class InsertGroupsActivity extends Activity {

    GroupsInsert.Result insertGroup(String title, Account account) {
        return ContactsFactory.create(this).groups().insert().group(title, account).commit();
    }

    GroupsInsert.Result insertGroups(List<NewGroup> groups) {
        return ContactsFactory.create(this).groups().insert().groups(groups).commit();
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/groups/kotlin/InsertGroupsActivity.kt
package contacts.sample.cheatsheet.groups.kotlin

import android.accounts.Account
import android.app.Activity
import contacts.core.Contacts
import contacts.core.entities.NewGroup
import contacts.core.groups.GroupsInsert

class InsertGroupsActivity : Activity() {

    fun insertGroup(title: String, account: Account): GroupsInsert.Result =
        Contacts(this).groups().insert().group(title, account).commit()

    fun insertGroups(groups: List<NewGroup>): GroupsInsert.Result =
        Contacts(this).groups().insert().groups(groups).commit()
}